hw/ip/flash_ctrl/dv/flash_ctrl_sim_cfg.hjson

```diff
@@ -54,6 +54,11 @@
     {
       name: flash_ctrl_smoke
       uvm_test_seq: flash_ctrl_smoke_vseq
+      reseed: 5
```

Comment:
not sure why reduce reseed to 5, a bigger number may help to stabilize it quickly

Reply:
The rand ops test runs the same test with more relaxed knobs. I chose to rely on that being solid instead. I felt 5 for the sanity was sufficient, given that we are already running too many jobs.